packages/shorebird_cli/lib/src/commands/patch/ios_framework_patcher.dart

```diff
@@ -51,6 +52,30 @@ class IosFrameworkPatcher extends Patcher {
   @override
   double? get linkPercentage => lastBuildLinkPercentage;
 
+  /// The value of --split-debug-info-path if specified.
+  String? get splitDebugInfoPath =>
+      argResults[CommonArguments.splitDebugInfoArg.name] as String?;
+
+  // TODO(felangel): make this dynamic based on the platform and arch.
+  /// The name of the split debug info file.
+  static const splitDebugInfoFileName = 'app.ios-arm64.symbols';
+
+  /// The additional gen_snapshot arguments to use when building the patch with
+  /// `--split-debug-info`.
+  List<String> get splitDebugInfoArgs {
+    return splitDebugInfoPath != null
+        ? [
+            '--dwarf-stack-traces',
+            '--resolve-dwarf-paths',
+            '''--save-debugging-info=${saveDebuggingInfoPath(splitDebugInfoPath!)}''',
```

Comment:
Did you mean '''

Reply:
Nope, I used the ''' explicitly to maintain the formatting